src/views/Log.vue

```diff
@@ -264,34 +295,59 @@ export default {
     }
   },
 
+  emits: [
+    updateInitialOptionsEvent,
+  ],
+
   props: {
-    initialOptions: {
-      type: Object,
-      required: false,
-      default: () => {}
-    }
+    initialOptions,
   },
```

Comment:
None of the other views have an initialOptions prop - is this a solution that can be rolled out for table, tree and graph views as well?

Reply:
Yes, the `initialOptions` prop can be added to them to allow storing their view states